### ISSUE TITLE
Why is reindex in Linear?

### ISSUE THREAD

**Issue by nducros** [openspyrit/spyrit#258]
https://spyrit.readthedocs.io/en/master/_autosummary/spyrit.core.meas.Linear.html

I would expect the method in child classes

**Comment by romainphan** [comment]
you mean in the base class ?

**Comment by nducros** [comment]
Yes. I would expect this "below" Linear (e.g., Hadam*), when introducing subsampling makes sense. What do you think?
